e2e/features/requestToken/RequestToken.feature

```diff
@@ -1,16 +1,32 @@
 Feature: Request Token
   Background: Add an account and navigate to wallet
     Given I add an account with passphrase "peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready" password "Password1$" name 'test_acc'
-    Then I go to page "wallet"
-
-  Scenario: Request token should generate a copy link
-    Given I click on a button with text "Request"
+    And I go to page "wallet"
+    And I wait for "2 seconds"
+    And I click on a button with text "Request"
     Then I should see "Request tokens"
     And "lisk_mainchain" should be selected in "Recipient application" dropdown
     And "Lisk" should be selected in "Token" dropdown
     And button with text 'Copy link' should be disabled
+    And Element 'qrContainer' should contain class 'disabled'
+
+  Scenario: Request token should not generate a copy link if amount is 0
+    Given I type "0" in "amount"
```

Comment:
👁️ we had a feature to allow users to send `0` amount with some message. 

The use case was for exchanges or something to do a check transaction or just to use data field for communication in network.

Good that we found the issue with e2e test 😉

Reply:
Yea, we seem to have a check for this specifically 
<img width="390" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/7137ce50-efa1-4984-83dc-0ab46da535f8">

I will change the test to, should be able to create a copy link with 0 😆

Reply:
I created this issue for this. https://github.com/LiskHQ/lisk-desktop/issues/5326

Reply:
But while it makes sense to send 0 amount with message, I don't think it makes sense to request 0 amount. Users who request usually need tokens to carry out transactions so they have less reason to request 0 amount. If a new user were to request 0 amount and receive it, they'd still be unable to send 0 amount in a transaction because they wouldn't have sufficient transaction fees

Reply:
@ikem-legend but it is for the scenario Manu wrote about earlier, exchange use case.

It should be allowed to generate a request transaction with 0 amount but just a message. This request link can someone click to to just send me a message

Reply:
I get the scenario but I'm just not sure it requires changes on the request end, only because of how less likely it might be. The exchanges or similar accounts who need to do such transactions can do them straight without needing to request. The point of the request token feature, in my opinion, is for users to actually get tokens for transactions so requesting 0 tokens just to make it faster for another user to send them a message seems less likely.

Reply:
I can see this point as well, which is why i didnt want to make any changes regarding that in this PR. Right now it works as you describe